Feature/author header photo

aleesteele opened the pull request:
I'm PRing this anyway due to time: but I'm having trouble making the opacity of the image because of the radial gradient on the background image of the page header. If it's set to a lower `z-index`, the opacity of the icon is `100%` but, but the header no longer has the gradient.

Other notes:
- Two conditional includes were made, above the header and below, that appear and disappear depending on the window size.
- Some refactoring to make more modular.
- Photo appears at top of `page-header` and `author-block`, and use the same component. Editing the color matrix in the include changes the tint.
- Photo tint/style is in its own component folder, so the color/size/photo can be edited later
- Each group was listed as a different class, instead of under a `.page-header` parent class because styles had been conflicting, ie applied universally to both elements

jnschrag commented on assets/_sass/components/_author-photo-tint.scss:
```diff
@@ -0,0 +1,32 @@
+    width: $image-size-header-mobile;
+    height: $image-size-header-mobile;
+
+    }
```
This is an extra `}`.

aleesteele commented:
- moved variables in scss to `variables` file so that they're served throughout the project, with standard: `$size-author-image-header-desktop`, `$size-author-image-header-mobile`, and 
`$size-author-image-block`
- made position of photo more scalable with `calc(#{$image-size-header-desktop} / -2)`, so that the header is always placed halfway between the header & content (the negative value of half the header-height)
- renamed include to something more modular: `author-photo-tint.html`
